index.html

```diff
@@ -202,97 +247,66 @@ <h2 class="hello2">About <br> Myself</h2>
-              <li>
-                <a href=""
-                  ><img
-                    src="images/linkedin.png"
-                    alt="Linkedin"
-                /></a>
-              </li>
-              <li>
-                <a href=""
-                  ><img src="images/sash.png" alt="M"
-                /></a>
-              </li>
-              <li>
-                <a href=""
-                  ><img src="images/twitter.png" alt="Github"
-                /></a>
-              </li>
-              <li>
-                <a href=""
-                  ><img src="images/M.png" alt="Peace"
-                /></a>
-              </li>
-            </ul>
-             
-            <div class="resumeButton" id="aboutResume">
+            <h4>LET’S CONNECT</h4>
+            <ul class="flex item-center">
+            <li class="icon-tag">
+              <a href=""><img src="images/github.png" alt="Twitter" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href=""><img src="images/linkedin.png" alt="Linkedin" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href=""><img src="images/sash.png" alt="M" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href=""><img src="images/twitter.png" alt="Github" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href=""><img src="images/sash.png" alt="Peace" /></a>
+            </li>
+          </ul>
+          <div class="resumeButton" id="aboutResume">
               <button type="button">Get my resume</button>
             </div>
-          </div>
-        </div>
+          
         </div>
+             
+            
 
-        <div class="about2">
-          <div class="lan-list">
-            <ul>
-              <li>
+                <div class="about2">
                 <div class="languages">
                   <h5>Languages</h5>
                   <img class="arrow" src="images/arrow.png" alt="" />
                 </div>
                 <ul class="aboutList2">
-                  <li>
-                    <div class="Q">
+                  <li class="Q">
                       <img src="images/java.png" alt="" />
-                      <p>JavaScript</p>
-                    </div>
+                      <p class="Q-text">JavaScript</p>
                   </li>
-                  <li>
-                    <div class="Q">
+                  <li class="Q">
                       <img src="images/html.png" alt="" />
-                      <p>HTML</p>
-                    </div>
+                      <p class="Q-text">HTML</p>
+                
                   </li>
-                  <li>
-                    <div class="Q">
+                  <li class="Q">
                       <img src="images/css.png" alt="" />
-                      <p>CSS</p>
-                    </div>
+                      <p class="Q-text">CSS</p>
                   </li>
                 </ul>
```

Comment:
- [ ] I can see that you are almost there. As requested by two previous reviewers, I have realized that I was also having the same issue of the JavaScript and CSS icons not showing. On further inspection, I realized that the problem was caused by a wrong file path to the images. Below is the correct one.

```
                <ul class="aboutList2">
                  <li class="Q">
                      <img src="./images/Java.png" alt="" />
                      <p class="Q-text">JavaScript</p>
                  </li>
                  <li class="Q">
                      <img src="images/html.png" alt="" />
                      <p class="Q-text">HTML</p>
                
                  </li>
                  <li class="Q">
                      <img src="./images/Css.png" alt="" />
                      <p class="Q-text">CSS</p>
                  </li>
                </ul>
```
**Note:** I believe you are using the Windows OS which is not always strict about file paths. However, on Unix/ mac systems and also when you deploy to Github pages or netlify, wrong file paths will also not show.

- [ ] There is also an extending horizontal line below the languages section which is supposed to be a little faint and does not extend till the end of the page. Kindly fix this.

**This is how it looks like:**

![image](https://user-images.githubusercontent.com/86318284/194719347-b4c55f6b-8087-49a8-a695-157f6c72c1bd.png)

**This is the correct design:**

![image](https://user-images.githubusercontent.com/86318284/194719365-07ebcbc3-d81b-447c-b213-e1a29c0763c8.png)